### ISSUE TITLE
Vertical Digger - Some players cannot change modes or reset the digger

### ISSUE THREAD

**Issue by tolly7654** [Ellpeck/ActuallyAdditions#750]
Title says it all

For some unknown reason, some players are completely unable to toggle any settings on the vertical digger (resetting Y coordinate and changing modes). This has happened to a small bunch of players on our server, and nothing comes up in the logs related to Actually Additions (or anything for that matter).

We've done some testing with the affected players, and they can use it fine on single player and their own test server, so we're not sure what's going on. Any help would be appreciated.

**Comment by Ellpeck** [comment]
Does reconnecting fix it?

**Comment by tolly7654** [comment]
Reconnecting does not fix the issue. We told the players to reinstall their packs and now that they have, they tried again and their issue still persists.

**Comment by tolly7654** [comment]
Okay, we did some further investigating, and it turns out it was due to an implementation of Bungeecord that we were all using. Sorry for making you guys panic, it all works fine.

**Comment by kaal-1** [comment]
It would be nice if you could post what you did to fix it.. This is a big problem since bungeecord is extensively used in almost every network nowadays.. And not only the vertical digger UI doesn't work, no other buttons/UIs do. That makes a big part of the mod unusable (ranged collectors, double crusher auto separate thingie, void sack, traveller's sack)..

**Comment by tolly7654** [comment]
With Bungeecord, we had it so we had a "lobby" that could be connected to via one address with any client from version 1.7.10 to 1.11.2. With this, we then used bungee to redirect people to servers, and this "indirect" connection to the modpack server caused this behavior. We fixed this by letting people know what the direct connection address to the server was. (For example, people would connect to hub.myserver.com and use /server or BungeePortals to join a new server with an IP of beyond.myserver.com. Making people connect directly to beyond.myserver.com fixed this issue).
